README.md

```diff
@@ -94,14 +95,33 @@ other manner of problems.  Caveat emptor.
 
 ## Getting Started
 
-If you want to get started with your own project immediately, you can use our
-`cargo risczero` tool to create your own repository. See
-[its README](cargo-risczero-readme) for installation and usage instructions.
+First, clone this repository and [install rust](install-rust).
+
+To run a simple RISC Zero program, navigate to the `examples/factors` directory
+and run `cargo run`:
+```
+cd examples/factors
+cargo run -r
+```
+This should produce the output `I know the factors of 391, and I can prove it!`.
+Congratulations, you've produced your first RISC Zero zero-knowledge proof! You
+can find more examples in this [examples folder](examples).
```

Comment:
I like that as an eventual plan, but it doesn't work right now and the changes to make it work aren't trivial. (For one thing, the examples `toml` files point to `risc0-zkvm` by relative path, so running something like `cargo risczero new test-example --template risc0/examples/factors` breaks. Also, if we want a no-clone approach we need tooling for downloading the example files, since that `risc0/examples/factors` mentioned in the previous command refers to the examples folder in the cloned repo.) All of which means I don't think we should move away from cloning the repo before the 0.14 release.

I could swap the order of the "To run a simple RISC Zero program" and the "To start your own project" sections, if you think that's better -- personally the current order feels more intuitive to me, but that's just a preference, not a strong opinion.

Reply:
I pushed a commit to this PR w/ the swapped order -- we can keep or revert.